release/release_guide.md

```diff
@@ -0,0 +1,570 @@
+<!--
```

Comment:
I have re-used the release guide when hudi was in incubation stage
https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=135859726